kv/txn_test.go

```diff
@@ -287,58 +278,53 @@ func TestTxnDBUncertainty(t *testing.T) {
 // node would lead to thousands of transaction restarts and almost certainly a
 // test timeout.
 func TestUncertaintyRestarts(t *testing.T) {
-	{
-		db, eng, clock, mClock, _, stopper, err := createTestDB()
+	s := createTestDB(t)
```

Comment:
This bit of the diff here and below is bogus--just removed vestigial bracketed scope which changed indenting.